portfolio/src/main/java/com/google/sps/data/LoginInfo.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+
+/**
+ * Class representing each comment object
+ */
+public class LoginInfo {
```

Comment:
I would recommend looking into AutoValue - https://github.com/google/auto/blob/master/value/userguide/index.md

Less boilerplate and less error prone and immutability

Reply:
I'm a little confused. The walkthroughs didn't mention any issues with user data, and they also say to deploy the changes once they're approved. Did I implement the user data component differently to how it was intended? As best I can tell I was working off the provided example code.

Reply:
(comments mixed up) AutoValue suggestion is for LoginInfo class.

I looked at the walkthrough for Authentication and those are the instructions. I am seeking further clarification. For now LGTM